docs/samples/git-setup/policytemplates/version_4.Y.Z/sno-ran-du/sno-ran-du-pg-v4-Y-Z-v4-full-DU.yaml

```diff
@@ -0,0 +1,376 @@
+          nodeSelector:
+            node-role.kubernetes.io/master: ''
+    # PTP CONFIG
+    - path: source-crs/PtpOperatorConfig-SetSelector.yaml
+      complianceType: "mustonlyhave"
+      patches:
+      - spec:
+          daemonNodeSelector:
+            node-role.kubernetes.io/worker: ""
+    - path: source-crs/PtpConfigSlave.yaml
+      openapi:
+        path: sno-ran-du/schema.openapi
+      patches:
+        - metadata:
+            name: du-ptp-slave
+          spec:
+            profile:
+            - interface: '{{hub fromConfigMap "" (printf "%s-pg" .ManagedClusterName) "ptpcfgslave-profile-interface" hub}}'
+              name: slave
+              phc2sysOpts: -a -r -n 24
+              ptp4lOpts: -2 -s --summary_interval -4
+            recommend:
+            - match:
+              - nodeLabel: node-role.kubernetes.io/master
+              priority: 4
+              profile: slave
+    # SRIOV OPERATOR CONFIG
+    - path: source-crs/SriovOperatorConfig-SetSelector.yaml
+      complianceType: musthave
+      patches:
+      - spec:
+          configDaemonNodeSelector:
+            node-role.kubernetes.io/master: ""
+          disableDrain: true
+          logLevel: 0
+    # LOCAL STORAGE CONFIG
+    - path: source-crs/StorageLV.yaml
+      patches:
+      - spec:
+          storageClassDevices:
+          - storageClassName: "example-storage-class-1"
+            volumeMode: Filesystem
+            fsType: xfs
+            devicePaths: '{{hub fromConfigMap "" (printf "%s-pg" .ManagedClusterName) "storagelv-devicePaths-1" | toLiteral hub}}'
+    - path: source-crs/StorageClass.yaml
+      patches:
+      - metadata:
+          name: example-storage-class-2
+    # CLUSTER LOGGING
+    - path: source-crs/ClusterLogForwarder.yaml
+      patches:
+      - spec:
+          $patch: replace
+          outputs: '{{hub fromConfigMap "" (printf "%s-pg" .ManagedClusterName) "cluster-log-fwd-outputs"  | toLiteral hub}}'
+          pipelines: '{{hub fromConfigMap "" (printf "%s-pg" .ManagedClusterName) "cluster-log-fwd-pipelines"  | toLiteral hub}}'
+          filters: '{{hub fromConfigMap "" (printf "%s-pg" .ManagedClusterName) "cluster-log-fwd-filters"  | toLiteral hub}}'
+          serviceAccount:
+            name: collector
+    # TUNED CONFIG
+    - path: source-crs/TunedPerformancePatch.yaml    
+      patches:
+      - spec:
+          recommend:
+            - machineConfigLabels:
+                machineconfiguration.openshift.io/role: master
+              priority: 19
+              profile: performance-patchh
+          profile:
+            - name: performance-patch
+              data: |
+                [main]
+                summary=Configuration changes profile inherited from performance created tuned
+                include=openshift-node-performance-openshift-node-performance-profile
+                [bootloader]
+                cmdline_crash=-tsc=nowatchdog
```

Comment:
Delete

Reply:
Do we need to override the default `profile`? 🤔 The ztp git example is just using the default profile values.

Reply:
No need to override the source cr here